docs/Manual/03 Schedules.md

```diff
@@ -16,39 +16,40 @@ for i in range(16):
         for k in range(11):
             C[i, j] += A[i, k] * B[k, j]
 ```
-In other words, each of the logical pseudo-code loops in `nest` becomes an actual for-loop in the default schedule.
+In other words, each of the logical pseudo-code loop in `nest` become an actual for-loop in the default schedule. 
 
-We can now transform this schedule in various ways. These transformations merely change the order of the loop iterations, and therefore preserve the logic defined in `nest`. Note that transforming `schedule` does not modify `nest`, and in fact, we could generate additional independent schedules by calling `nest.create_schedule()` multiple times.
```

Comment:
Let's revert to the "We ..." conversational tone for the text. Other editorial modifications are ok.